pyproject.toml

```diff
@@ -1,13 +1,16 @@
 [build-system]
-requires = ["poetry-core>=1.1.0a6"]
+requires = ["poetry-core>=1.2.0b2"]
```

Comment:
change to using 1.2 rather than b2 as 1.2 is out now.

Reply:
Looks like poetry just updated to [1.2.2](https://python-poetry.org/blog/category/releases/). Will update and test locally before pushing a new commit.